### PR TITLE
feat(docs): Add Gauge.md

### PR DESCRIPTION
**Pull request by jbrundage**
Add enhancements to Gauge.ts

Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by jbrundage** [comment]
image

**Comment by jbrundage** [comment]
image

File: packages/chart/src/Gauge.ts [hunk 1]
```diff
@@ -335,3 +340,19 @@ export class Gauge extends SVGWidget {
...
 }
 Gauge.prototype._class += " chart_Gauge";
+
+export interface Gauge {
+    tickColor(): string;
+    tickColor(_: string): this;
+    emptyColor(): string;
+    emptyColor(_: string): this;
+    colorDomain(): any;
+    colorDomain(_: any): this;
```
Comment (GordonSmith) [inline packages/chart/src/Gauge.ts]: Typing is wrong

File: packages/chart/src/Gauge.ts [hunk 2]
```diff
@@ -335,3 +340,19 @@ export class Gauge extends SVGWidget {
...
+    colorRange(): any;
+    colorRange(_: any): this;
```
Comment (GordonSmith) [inline packages/chart/src/Gauge.ts]: Typing is wrong

File: packages/chart/src/Gauge.ts [hunk 3]
```diff
@@ -72,6 +63,9 @@ export class Gauge extends SVGWidget {
     protected _bottomText: any;
     protected _tooltipG: any;
     protected _mainTooltip: any;
+    protected _colorScale: any = scaleLinear<string, string>()
```
Comment (GordonSmith) [inline packages/chart/src/Gauge.ts]: Remove "any" type (if you can)

**Pull request merged**